### PR TITLE
HBW-191 Add tests for searching in claiming table

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: spec/hbw/features/tasks/search_spec.rb [hunk 1]
```diff
@@ -0,0 +1,70 @@
+feature 'Check searching in table with tasks', js: true do
+  before(:each) do
+    user = FactoryBot.create(:user)
+
+    signin(user.email, user.password)
+    expect(page).not_to have_content 'Sign in'
+    expect(page).to     have_content 'Orders list'
+
+    ajax { click_on 'Tasks' }
```
Comment (TimAle) [inline spec/hbw/features/tasks/search_spec.rb]: we have `click_and_wait` helper method

File: spec/hbw/features/tasks/search_spec.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    expect_widget_presence
+    expect(page).to have_content 'Open tasks'
+  end
+
+  describe 'search field rendered' do
+    it 'with too short search query' do
+      fill_search_field('te')
+      expect_widget_presence
```
Comment (TimAle) [inline spec/hbw/features/tasks/search_spec.rb]: this check is not related to search results or task list at all

File: spec/hbw/features/tasks/search_spec.rb [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+    it 'with valid search request' do
+      expect(tasks_table_content).to eq(
+        [
+          ['Medium', 'Check test form', ' Test name', 'No description', 'expired'],
+          ['High', 'Check test form', ' Test name', 'Some test description', '30 Jun 2016']
+        ]
+      )
+
+      fill_search_field('description')
```
Comment (TimAle) [inline spec/hbw/features/tasks/search_spec.rb]: Not the best example. We have placeholder `No description` which contains word `description` but we can't see this row after searching. This is a bit confusing, let's choose other query.

File: spec/hbw/features/tasks/search_spec.rb [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+      expect(search_field_text).to eq('description')
+
+      clear_search_field
+
+      expect(search_field_text).to eq('')
+      expect(tasks_table_content).to eq(
+        [
+          ['Medium', 'Check test form', ' Test name', 'No description', 'expired'],
+          ['High', 'Check test form', ' Test name', 'Some test description', '30 Jun 2016']
```
Comment (TimAle) [inline spec/hbw/features/tasks/search_spec.rb]: well, you just checked that nothing has been changed :)

File: app/views/tasks/index.html.haml [hunk 5]
```diff
@@ -1 +1,2 @@
+- content_for(:title, t('.title'))
```
Comment (TimAle) [inline app/views/tasks/index.html.haml]: good catch!

File: spec/hbw/features/tasks/search_spec.rb (hunk 4 above)
Comment (mxsmirnoff) [reply spec/hbw/features/tasks/search_spec.rb]: Thats the point. After clearing search field everything has to be as before search.

Comment (TimAle) [reply spec/hbw/features/tasks/search_spec.rb]: But you should check that clear cross will change state when clicked.

File: spec/hbw/features/tasks/search_spec.rb [hunk 6]
```diff
@@ -0,0 +1,69 @@
...
+    expect(page).to     have_content 'Orders list'
+
+    click_and_wait 'Tasks'
+    expect_widget_presence
+    expect(page).to have_content 'Open tasks'
+  end
+
+  describe 'search field rendered' do
+    it 'with too short search query' do
+      fill_search_field('te')
```
Comment (TimAle) [inline spec/hbw/features/tasks/search_spec.rb]: check that results are the same as initial
